pkg/collect/logs.go

```diff
@@ -89,6 +89,7 @@ func listPodsInSelectors(ctx context.Context, client *kubernetes.Clientset, name
 
 	listOptions := metav1.ListOptions{
 		LabelSelector: serializedLabelSelector,
+		FieldSelector: "status.phase!=Failed",
```

Comment:
Consider using functions provided in `k8s.io/apimachinery/pkg/fields`: fields.OneTermNotEqualSelector("status.phase", corev1.PodFailed).String()`  and also passing these in as arguments to the function when needed

Reply:
@divolgin ty for the correction!